packages/block-editor/src/components/global-styles/advanced-panel.js

```diff
@@ -30,7 +30,7 @@ export default function AdvancedPanel( {
 		} );
 		if ( cssError ) {
 			const [ transformed ] = transformStyles(
-				[ { css: value } ],
+				[ { css: newValue } ],
```

Comment:
@afercia @youknowriad this was a bug. We were passing the existing `value` (which is composed of `{css: 'existing'}`) instead of the recently changed rules (this is the on change handler).

With this change, we're passing the correct value. If you revert it in your local environment, you'll see that `[object Object]` gets passed whenever you apply a change after blurring the field with an invalid CSS.

For the sake of expediting this change, which is breaking a couple of places, I'd say we should keep it. If we want to remove it and apply this change in a different PR, fine by me.